ci/cpu/build.sh

```diff
@@ -118,4 +118,4 @@ fi
 ################################################################################
 
 gpuci_logger "Upload conda packages"
-source ci/cpu/upload.sh
```

Comment:
Removing the newline at the end of this file seems like an inadvertent change

Reply:
Right. I modified this file earlier when I was debugging and when I reset it it looks like I removed the new line